ndnsd/communication/sync-adapter.cpp

```diff
@@ -15,28 +15,31 @@
  *
  * You should have received a copy of the GNU Lesser General Public License along with
  * NDNSD, e.g., in COPYING.md file.  If not, see <http://www.gnu.org/licenses/>.
+
+  @ most part of sync-adapter code is taken from NLSR/communication/
+
  **/
 
 #include "sync-adapter.hpp"
-// #include "logger.hpp"
+#include <iostream>
 
-// INIT_LOGGER(SyncProtocolAdapter);
+INIT_LOGGER(SyncProtocolAdapter);
 
 namespace ndnsd {
 
 const auto FIXED_SESSION = ndn::name::Component::fromNumber(0);
 
 SyncProtocolAdapter::SyncProtocolAdapter(ndn::Face& face,
-                                         int32_t syncProtocol,
+                                         uint8_t syncProtocol,
                                          const ndn::Name& syncPrefix,
                                          const ndn::Name& userPrefix,
                                          ndn::time::milliseconds syncInterestLifetime,
                                          const SyncUpdateCallback& syncUpdateCallback)
- : m_syncProtocol(syncProtocol)
- , m_syncUpdateCallback(syncUpdateCallback)
+  : m_syncProtocol(syncProtocol)
+  , m_syncUpdateCallback(syncUpdateCallback)
 {
   if (m_syncProtocol == SYNC_PROTOCOL_CHRONOSYNC) {
-    NDN_LOG_DEBUG("Using ChronoSync");
+    NDNSD_LOG_DEBUG("Using ChronoSync");
```

Comment:
These should change to NDN_LOG_DEBUG, once you remove custom log file.

Reply:
done.